File: hackerrank/templates/template.py
#!python3.5

import sys
import io

TEST = """"""

def solve(reader):
  # test_cases=reader.readline()
  # for i in range(int(test_cases)):
  #   a, b, c = [ int(v) for v in reader.readline().strip().split() ]
  pass

def main():
  # for testing purposes
  if len(sys.argv) == 2 and sys.argv[1] == "test":
    solve(io.StringIO(TEST))
    return
  solve(sys.stdin)

if __name__ == "__main__":
  main()

File: hackerrank/templates/template.rb
require 'stringio'

TEST = ""

def solve(input)
  #string = input.gets.chomp.downcase
end

def main()
  # for testing purposes
  if ARGV.length == 1 && ARGV[0] == "test"
    solve(StringIO.new(TEST))
    return
  end

  solve($stdin)
end

if __FILE__ == $0
  main()
end
